### PR TITLE
Update optimizer

### LINKED ISSUE #20: Compatibility with Adam
**Is your feature request related to a problem? Please describe.**
Currently, standard gradient ascent is used as an optimizer. Adding compatibility for Adam could lead to better results.

**Describe the solution you'd like**
Add other optimisers such as Adam for generating feature visualizations.

### PR DESCRIPTION
**Pull request by fuzhanrahmanian**
Adding support for Adam optimizer
resolves #20

### CODE REVIEW

File: luna/featurevis/featurevis.py [hunk 1]
```diff
@@ -13,17 +13,28 @@
...
     """object for generalizing optimization parameters.
 
     Args:
         iterations (number): how many iterations to optimize for.
         learning_rate (number): update amount after each iteration.
+        optimizer (class): optimizer (only Adam is supported)
     """
 
-    def __init__(self, iterations, learning_rate):
+    def __init__(self, iterations, learning_rate, optimizer=None) -> None:
```
Comment (Sparkier) [inline luna/featurevis/featurevis.py:25]: I don't think you need the `-> None` do you?

**Review by Sparkier — APPROVED** [review]

**Review by Sparkier — APPROVED** [review]
